lib/Parser/Parse.cpp

```diff
@@ -13933,25 +13920,25 @@ void ParseNode::Dump()
 }
 #endif
 
-DeferredFunctionStub * BuildDeferredStubTree(ParseNode *pnodeFnc, Recycler *recycler)
+DeferredFunctionStub * BuildDeferredStubTree(ParseNodeFnc *pnodeFnc, Recycler *recycler)
```

Comment:
Good to see; I have this same change in my working branch. Looks like it'll merge cleanly.